tests/test_blueprint_copy.py

```diff
@@ -66,3 +66,29 @@ def handle_request(request):
 
     _, response = app.test_client.get("/version6/page")
     assert "Hello world!" in response.text
+
+
+def test_bp_copy_with_route_overwriting(app: Sanic):
+    bpv1 = Blueprint("bp_v1", version=1)
```

Comment:
I'd throw something like `url_prefix` on the test just so that it shows some constant value is being copied and still applied.

Reply:
Added